tcl/org.testeditor.tcl.dsl.ui.tests/resources/test.aml

```diff
@@ -0,0 +1,70 @@
+package SwingDemo
```

Comment:
There is missing a lot. I would have to extend it to my needs. Do you think it makes sense?